panel/depends.py

```diff
@@ -25,8 +29,8 @@ def param_value_if_widget(arg):
     return arg
 
 
-def depends(*args, **kwargs):
-    updated_args = [param_value_if_widget(a) for a in  args]
-    updated_kwargs = {k: param_value_if_widget(v) for k, v in kwargs.items()}
+def depends(*args, throttled=False, **kwargs):
```

Comment:
Little bit worried that this will clash with a user provided keyword called throttled, currently don't have a better suggestion though.

Reply:
I am not the biggest fan of it either. Maybe it could be changed to be sat globally by something like `pn.extension(slider_wait=True)`?

Reply:
I do think you want control over this per instance and technically you can already achieve this with `pn.depends` by passing in the `widget.param.value_throttled` param explicitly, so maybe in that case it's just a matter of properly documenting that?

Reply:
You are right that their need to be proper documentation. What about a list of reserved keywords for `pn.interact` and `pn.depends`. I properly need a little help in finding where I should document it… I would think that it needs to be documented in a notebook file(s), can you help me here?

I was also looking at the error message for watch and I think there should be some overlap between the error messages for the keyword `watch` and a keyword like `throttled`. I have attached a pdf file with the different error messages for `watch` and `throttled` for `pn.interact` and `pn.depends`.